tools/build_file_generator/JavaClassAnalyzer.java

```diff
@@ -20,14 +20,125 @@
 import com.google.startupos.tools.build_file_generator.Protos.Import;
 import com.google.startupos.tools.build_file_generator.Protos.JavaClass;
 import java.io.IOException;
+import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 import javax.inject.Inject;
 
 public class JavaClassAnalyzer {
   private FileUtils fileUtils;
 
+  // TODO: Come up with a better way to get this list
+  private List<String> javaLangClasses =
+      Arrays.asList(
```

Comment:
Is this list sorted by any logic? If not, how about sorting it alphabetically?

Reply:
The list wasn't sorted by any logic. It's sorted alphabetically now.

Reply:
Thanks!